### DISCUSSION TITLE
Big number issue on return values (C, C#)

### DISCUSSION THREAD

**Discussion by Inerska** [PrismJS/prism#2941] [Q&A] [answered]
Hello, just getting started with PrismJS for my static blog.
When I put a code block, especially with a return keyword, the value is very big.

Is it normal ? Can we disable it?

![image](https://user-images.githubusercontent.com/50674490/121812708-410dfc00-cc69-11eb-86a9-6273c74060bb.png)

**Comment by RunDevelopment** [comment, marked as answer]
This is definitely not normal. 

The problem is most likely something caused by your style sheets. Please inspect the number element and look at which style sheet changes the font size.

Looking at your screenshot, the number also gets a white background and some padding. It's not just the font size.

**Reply by Inerska** [reply]
Oh, I'm dumb, that's my CSS framework, sorry for the bother.
Thanks.
